src/convertToObject.js

```diff
@@ -16,8 +16,8 @@
 function convertToObject(sourceString) {
   const result = {};
 
-  const modifiedStr = sourceString.split(';');
-  const mod2 = modifiedStr.map(x => x.split(':'));
+  const modifiedStr = sourceString.split(';')
+    .map(x => x.split(':'));
```

Comment:
I dont understand what is this ?  What is before this (.map.....) ???

Reply:
it`s same as `const modifiedStr = sourceString.split(';').map(x => x.split(':'));`